modules/custom_operations/user_ie_extensions/tokenizer/python/README.md

```diff
@@ -6,40 +6,32 @@
   - Fast tokenizers based on Wordpiece and BPE models
   - Slow tokenizers based on SentencePiece model file
 - Combine OpenVINO models into a single model
-- Add greedy decoding pipeline to text generation model 
+- Add greedy decoding pipeline to text generation model
 
 ## Installation
 
-1. Build the extension with the `-DCUSTOM_OPERATIONS="tokenizer"` flag: [instruction](../../../README.md#build-custom-openvino-operation-extension-library)
+1. Install [OpenVINO Runtime for C++](https://docs.openvino.ai/latest/openvino_docs_install_guides_install_dev_tools.html#for-c-developers).
 2. (Recommended) Create and activate virtual env:
 ```bash
 python3 -m venv venv
 source venv/bin/activate
 ```
-3. Go to `modules/custom_operations/user_ie_extensions/tokenizer/python` and run:
+3. Go to `modules/custom_operations` and run:
 ```bash
 # to use converted tokenizers or models combined with tokenizers
 pip install .
 # to convert tokenizers from transformers library
-pip install .[transformers] 
+pip install .[transformers]
 # for development and testing the library
-pip isntall -e .[all]
+pip install -e .[all]
 ```
 
-## Usage
-
-Set `OV_TOKENIZER_PREBUILD_EXTENSION_PATH` environment variable to `libuser_ov_extensions.so` file path
-or use `init_extension` function.
-
 ### Convert HuggingFace tokenizer
 
 ```python
 from transformers import AutoTokenizer
 from openvino import compile_model
-from ov_tokenizer import init_extension, convert_tokenizer, pack_strings
-
-
-init_extension("path/to/libuser_ov_extensions.so")
+from ov_tokenizer import convert_tokenizer, pack_strings
```

Comment:
`ov_tokenizer` should be renamed to `openvino_tokenizer` too

Reply:
and we need to rename the folder itself https://github.com/openvinotoolkit/openvino_contrib/tree/e4909171e279fd792ae609d4d15e5dbc0b0513ad/modules/custom_operations/user_ie_extensions/tokenizer/python/ov_tokenizer

Reply:
renamed to openvino_tokenizers